W-19719053-agent-fabric-stakeholder-feedback-on-first-draft-mk (#507)
mulesoft/docs-code-builder: modules/ROOT/pages/af-project-files.adoc

=== Exchange.json

> All agent network projects also have an `exchange.json` file. If you're familiar with Anypoint Platform, you've probably seen this file before. This file contains asset metadata that will be available in Anypoint Exchange after you publish your agent network assets.

Reviewer: ```suggestion
All agent network projects also have an `exchange.json` file. This file contains asset metadata available in Anypoint Exchange after publishing your agent network assets.
```

Reviewer: The user may or may not be familiar with Anypoint Platform. In any case, we provide the steps to complete the task, so it seems we may omit this information.

In this example, the Onboarding Broker orchestrates the process of onboarding a new hire. It orchestrates work with the IT, HR and security brokers, which also happen to be brokers in the same agent network.

> Assets, IT Stock, HR, Payroll, and Badges agents are what we call leaf agents. These aren't brokers but preexisting agents whose purpose isn't to coordinate work, but to act.

Reviewer: ```suggestion
Assets, IT Stock, HR, Payroll, and Badges agents are also called *leaf agents*. These aren't brokers but preexisting agents whose purpose isn't to coordinate work, but to act.
```

> ==== Llm-providers

Reviewer: ```suggestion
==== Llm-Providers
```

Author: lowercased in YAML
